system_files/desktop/shared/usr/lib/systemd/system/bazzite-hardware-setup.service

```diff
@@ -7,6 +7,7 @@ Before=systemd-user-sessions.service jupiter-biosupdate.service jupiter-controll
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=/usr/bin/bazzite-hardware-setup
+ConditionPathExists=!/etc/bazzite/hardware_setup_done
```

Comment:
I've been brainstorming a better place to put this but honestly I can't think of anywhere else it would belong

Reply:
If someone eventually makes a standard "universal blue" configuration directory, we can move that check file onto it. For now I guess this would be the best?